torch/export/_trace.py

```diff
@@ -1528,7 +1527,7 @@ def custom_getattribute(self, attr, *, original_getattr, attrs_to_proxy):
                                     proxy = get_proxy_slot(self, tracer).proxy
                                     inner_proxy = tracer.create_proxy(
                                         "call_function",
-                                        builtins.getattr,
+                                        torch.ops.aten._access_subclass_inner_tensor.default,
```

Comment:
hmm I think we also agreed to "const-propagate through any constant metadata" as part of the getattr changes